go/teams/service_helper.go

```diff
@@ -767,11 +770,15 @@ func loadUserVersionByUIDCheckUsername(ctx context.Context, g *libkb.GlobalConte
 		return keybase1.UserVersion{}, libkb.BadUsernameError{N: un}
 	}
 
+	uv := NewUserVersion(upak.Current.Uid, upak.Current.EldestSeqno)
+	if upak.Current.Status == keybase1.StatusCode_SCDeleted {
+		return uv, errUserDeleted
```

Comment:
Can we live with this for now and have a ticket to redo the `loadUserVersion*` helper functions later?

Reply:
Sure, it's fine with me.  Please make tickets for any planned future work.